stdlib/public/core/Set.swift

```diff
@@ -116,13 +116,12 @@
 /// collection wrapper instead of a set. To restore efficient set operations,
 /// create a new set from the result.
 ///
-///     let morePrimes = primes.union([11, 13, 17, 19])
-///
-///     let laterPrimes = morePrimes.filter { $0 > 10 }
-///     // 'laterPrimes' is of type Array<Int>
-///
-///     let laterPrimesSet = Set(morePrimes.filter { $0 > 10 })
-///     // 'laterPrimesSet' is of type Set<Int>
+///     let formatter = NumberFormatter()
+///     formatter.numberStyle = .spellOut
+///     let primesSpellOut = primes.map(NSNumber.init).compactMap(formatter.string)
+///     // 'primesSpellOut' is of type Array<String>
+///     let primesSpellOutSet = Set(primes.map(NSNumber.init).compactMap(formatter.string))
+///     // 'primesSpellOutSet' is of type Set<String>
```

Comment:
What do you think of sticking to simple standard library types? A user who would benefit from this example likely has not progressed to learning about NumberFormatter and NSNumber.

It does not have to a useful example; it merely has to demonstrate the return type of a function such as ‘map’ and how to get a Set back.